eng/Version.Details.xml

```diff
@@ -1,15 +1,130 @@
+    <Dependency Name="Microsoft.DotNet.Cli.CommandLine" Version="1.0.0-preview.19127.1">
+      <Uri>https://github.com/dotnet/cliCommandLineParser</Uri>
+      <Sha>0e89c2116ad28e404ba56c14d1c3f938caa25a01</Sha>
+      <RepoName>clicommandlineparser</RepoName>
+    </Dependency>
+    <Dependency Name="Microsoft.ApplicationInsights" Version="2.0.0">
+      <Uri>https://github.com/Microsoft/ApplicationInsights-dotnet</Uri>
+      <Sha>53b80940842204f78708a538628288ff5d741a1d</Sha>
+      <RepoName>application-insights</RepoName>
+    </Dependency>
+    <Dependency Name="XliffTasks" Version="0.2.0-beta-000048">
+      <Uri>https://github.com/dotnet/xliff-tasks</Uri>
+      <Sha>27d43b762aa6dac3a0a6ba48fe55000942d75c1c</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Cli.Runtime" Version="3.0.100-preview4.19164.1">
+      <Uri>https://github.com/dotnet/cli</Uri>
+      <Sha>204f425b6f061d0b8a01faf46f762ecf71436f68</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.FSharp.Compiler" Version="10.4.2-rtm-190302-01">
+      <Uri>https://github.com/microsoft/visualfsharp</Uri>
+      <Sha>d4e68aef7fa3507d38dd7fe4eca287fc40dd46ce</Sha>
+      <RepoName>fsharp</RepoName>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Sdk.Web" Version="3.0.100-preview4.19167.1">
+      <Uri>https://github.com/aspnet/websdk</Uri>
+      <Sha>b55d4f4cf22bee7ec9a2ca5f49d54ebf6ee67e83</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.TestPlatform.Build" Version="15.9.0-preview-20180807-05">
+      <Uri>https://github.com/Microsoft/vstest</Uri>
+      <Sha>df96f832f28de9798bf3a1b2d6b7933439e00ae4</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Dotnet.Toolset.Internal" Version="3.0.100-preview4.19167.1">
+      <Uri>https://github.com/dotnet/toolset</Uri>
+      <Sha>3165b2579582b6b44aef768c01c3cc9ff4f0bc17</Sha>
+    </Dependency>
+    <Dependency Name="core-sdk" Version="3.0.100-preview3">
+      <Uri>https://github.com/dotnet/core-sdk</Uri>
+      <Sha>118dd862c853ff39694449ac303f5d8ef7d11b24</Sha>
+    </Dependency>
+
+    <!-- external dependencies, not handled by Maestro/Arcade -->
+    <Dependency Name="Newtonsoft.Json" Version="9.0.1">
+      <Uri>https://github.com/dseefeld/Newtonsoft.Json</Uri>
+      <Sha>e43dae94c26f0c30e9095327a3a9eac87193923d</Sha>
+      <RepoName>newtonsoft-json</RepoName>
+    </Dependency>
+
+    <!-- not yet implemented -->
+    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="3.0.0-preview4-19168-01">
+      <Uri>https://github.com/aspnet/AspNetCore</Uri>
+      <Sha>5c6e1aa1d95bb79809c99736acb3b8abd36d08d4</Sha>
+      <SkipClone>True</SkipClone>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Sdk.Razor" Version="3.0.0-preview4.19167.2">
+      <Uri>https://github.com/aspnet/AspNetCore-Tooling</Uri>
+      <Sha>08651acc42d11510f7a9c47d47f9135efd1d1d12</Sha>
+      <SkipClone>True</SkipClone>
+    </Dependency>
+    <Dependency Name="dotnet-ef" Version="3.0.0-preview4.19167.2">
```

Comment:
Is this a new dependency?  What brought this in?

Reply:
Removed extra dependencies.